or-model/src/test/java/com/google/refine/history/HistoryEntryTests.java

```diff
@@ -34,27 +34,21 @@
 import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
-import com.google.refine.RefineTest;
-import com.google.refine.history.HistoryEntry;
+import com.google.refine.ProjectManager;
+import com.google.refine.ProjectManagerStub;
 import com.google.refine.model.Project;
 import com.google.refine.operations.OperationRegistry;
-import com.google.refine.operations.column.ColumnAdditionOperation;
+import com.google.refine.util.ParsingUtilities;
 import com.google.refine.util.TestUtils;
 
-public class HistoryEntryTests extends RefineTest {
+public class HistoryEntryTests {
 	
     public static final String fullJson = "{"
             + "\"id\":1533633623158,"
             + "\"description\":\"Create new column uri based on column country by filling 269 rows with grel:\\\"https://www.wikidata.org/wiki/\\\"+cell.recon.match.id\","
             + "\"time\":\"2018-08-07T09:06:37Z\","
-            + "\"operation\":{\"op\":\"core/column-addition\","
-            + "   \"description\":\"Create column uri at index 2 based on column country using expression grel:\\\"https://www.wikidata.org/wiki/\\\"+cell.recon.match.id\","
-            + "   \"engineConfig\":{\"mode\":\"row-based\",\"facets\":[]},"
-            + "   \"newColumnName\":\"uri\","
-            + "   \"columnInsertIndex\":2,"
-            + "   \"baseColumnName\":\"country\","
-            + "   \"expression\":\"grel:\\\"https://www.wikidata.org/wiki/\\\"+cell.recon.match.id\","
-            + "   \"onError\":\"set-to-blank\"}"
+            + "\"operation\":{\"op\":\"core/my-operation\","
```

Comment:
Because the previous version of this test made references to a specific operation and GREL expression. Those are features which are now defined in other Maven modules, which are not accessible in this scope. `HistoryEntry` is part of the core model of OpenRefine (`or-model`), on which GREL (`or-grel`) and workflow elements (`or-workflow`) depend. Therefore it is necessary to introduce a stub operation in this case. From a unit testing perspective, it also makes more sense to me since in this test we are not meant to assert anything about how a specific operation is serialized, but rather that the serialization of `HistoryEntry` relies on the serialization of the contained operation.